src/intervalbridge.jl

```diff
@@ -0,0 +1,54 @@
+# l ≤ ⟨a, x⟩ + α ≤ u
+# is transformed into
+# ⟨a, x⟩ + α ≥ l
+# ⟨a, x⟩ + α ≤ u
+struct SplitIntervalBridge{T}
+    lower::CR{MOI.ScalarAffineFunction{T}, MOI.GreaterThan{T}}
+    upper::CR{MOI.ScalarAffineFunction{T}, MOI.LessThan{T}}
+end
+function SplitIntervalBridge(instance, f::MOI.ScalarAffineFunction, s::MOI.Interval)
+    lower = MOI.addconstraint!(instance, f, MOI.GreaterThan(s.lower))
+    upper = MOI.addconstraint!(instance, f, MOI.LessThan(s.upper))
+    SplitIntervalBridge(lower, upper)
+end
+MOI.get(b::SplitIntervalBridge, ::MOI.NumberOfConstraints) = 0
```

Comment:
Please document what `MOI.get(::SplitIntervalBridge,...)` means, given that `SplitIntervalBridge` is not part of the MOI type hierarchy.